htdocs/js/components/StaticDataTable.js

```diff
@@ -10,7 +12,7 @@ StaticDataTable = React.createClass({
         // parameters of the form: func(ColumnName, CellData, EntireRowData)
         getFormattedCell: React.PropTypes.func
     },
-    componentDidMount: function () {
+    componentDidMount: function componentDidMount() {
```

Comment:
@driusan Yes, this is a compiled JS generated by Babel using ES2015 preset.

>The duplicated function names you're seeing here aren't an issue – it's perfectly valid to assign a named function to a variable, and it can make it easier to debug your code as debuggers will be able to show you that name instead of just listing it as an anonymous function.

- Explanation from **[here](http://stackoverflow.com/questions/35857279/babel-duplicated-function-names)**